tsl/test/expected/data_node.out

```diff
@@ -1291,9 +1291,31 @@ NOTICE:  database "db_data_node" already exists on data node, skipping
 NOTICE:  extension "timescaledb" already exists on data node, skipping
 ERROR:  [data_node_99]: cannot add the current database as a data node to itself
 \set ON_ERROR_STOP 1
+-- Test adding bootstrapped data node where extension owner is different from user adding a data node
+DROP DATABASE IF EXISTS data_node_6;
```

Comment:
You shouldn't need this drop; just drop the database at the end of the test.

Reply:
I've added this as a precaution since we reference `data_node_6 ` in some previous test. Anyways, seems that tests run fine without it as well.

Reply:
Yeah, but we've sort of decided on not reusing databases between tests and create new ones in each test (I prefix database names with the test name). This will allow the tests to be parallelized. Good that it works without, so suggest you remove it.